modules/release-notes/pages/relnotes.adoc

```diff
@@ -188,12 +188,68 @@ The Couchbase JIRA filter https://issues.couchbase.com/issues/?filter=19680[Couc
 [#common-vulnerabilities-exposures-700]
 === Common Vulnerabilities and Exposures
 
-This section lists common vulnerabilities and exposures that are fixed in this release. 
+This section lists common vulnerabilities and exposures that are fixed in this release.
 
 See https://www.couchbase.com/alerts[Couchbase Alerts] for the complete list of common vulnerabilities and exposures.
 
-* CVE-2021-35943 
+* CVE-2021-35943
 * https://nvd.nist.gov/vuln/detail/CVE-2021-23840[CVE-2021-23840]
 * https://nvd.nist.gov/vuln/detail/CVE-2019-10768[CVE-2019-10768]
 * https://nvd.nist.gov/vuln/detail/CVE-2021-3450[CVE-2021-3450]
-* https://nvd.nist.gov/vuln/detail/CVE-2021-3449[CVE-2021-3449]
\ No newline at end of file
+* https://nvd.nist.gov/vuln/detail/CVE-2021-3449[CVE-2021-3449]
+
+[#release-701]
```

Comment:
We list the latest release at the top of the page, so release 7.0.1 should be before 7.0.0

Reply:
I"ve changed the order accordingly. Thanks.